README.md

```diff
@@ -314,78 +299,196 @@ bash ./scripts/create-container-registry-secret.sh
 
 The *Docker Server* is the full qualified hostname of your container registry.
 
-#### Create a HDI container and a secret
+#### Create a HDI container / Service Manager Instance and a Secret
+
+This step is only required if you're using a BTP Trial account. If you're using a production or a free tier account then you can create HDI Container from Kyma directly by adding a [mapping to your Kyma namespace in your HANA Cloud Instance](https://blogs.sap.com/2022/12/15/consuming-sap-hana-cloud-from-the-kyma-environment/) and skip this step.
+
+##### Single Tenant
 
 ```
 bash ./scripts/create-db-secret.sh bookshop-db
 ```
 
-It will create a HDI container `bookshop-db` on your currently targeted Cloud Foundry space and creates a secret `bookshop-db` with the HDI container's credentials in your current Kubernetes namespace.
+##### Multi Tenant
 
-### Build
+```
+bash ./scripts/create-sm-secret.sh bookshop-sm
+```
+
+It will create a HDI container `bookshop-db` (single tenant) or a Service Manager `bookshop-sm` (multi tenant) instance on your currently targeted Cloud Foundry space and creates a secret `bookshop-db` (single tenant) or `bookshop-sm` (multi tenant) with the credentials in your current Kubernetes namespace.
+
+Make the following changes to your _`chart/values.yaml`_.
+
+##### Single Tenant
```

Comment:
What about combining both multi tenant and single tenant sections into one to only have 2 headings instead of 4?

Reply:
done